cromshell

```diff
@@ -1408,7 +1501,7 @@ if ${ISINTERACTIVESHELL} ; then
   checkForComplexHelpArgument $@
 
   # Get flags:
-  while getopts ":t:" opt ; do
+  while getopts ":t:xp" opt ; do
```

Comment:
Since these two flags only get processed by `execution-status-count` we don't need to check for them here.  

They can then be passed as arguments to the command rather than the high level script:

```
./cromshell execution-status-count -p -x a63db8dd-e34e-4156-923f-7d10a68e4cb8 5ea63275-6287-42b9-8be2-1460368a491a
```